test/integration/device/device_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package integration
+package device_test
```

Comment:
package device?

Reply:
@m1093782566 , As part of enhancing the integration test, we have identied the tests can be based on the feature supported by kubeedge. Hence the packages are now named as per the tests it is doing. Please confirm me whether i can keep the same package name or do you feel it needs to be chaged ?